homeassistant/components/alert/services.yaml

```diff
@@ -1,12 +1,24 @@
 toggle:
   description: Toggle alert's notifications.
   fields:
-    entity_id: {description: Name of the alert to toggle., example: alert.garage_door_open}
+    entity_id:
+      {
```

Comment:
This should not be here. There should just be the keys.

Reply:
Removed flow mapping using curly braces from all service files.

Reply:
Unfortunately, this is not enforceable.

Reply:
😢
What about the yaml lint? It's not configurable for this?

Reply:
Nope, in YAMLLint I can disable the braces rule (which does nothing) or specificy how braces are formatted.

Both cases result in the allowance of braces.

Reply:
<https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.braces>

Reply:
Ok. 😭